### PR TITLE
create Add and remove function to list

### PR DESCRIPTION
**Pull request by Reem-lab**
## Changes that I made here 👍 
 - set up the project with my webpack template 
 - Create the Tasks dynamically by input 
 - Add remove trash to remove tasks
 - Add completed and index to the task
 - save them in local storage
 - when you delete the item from the list it deleted from local storage also
 - no linter error
 - Add some styles to my list

### CODE REVIEW

File: src/createList.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import './style.css';
+
+const listContainer = document.querySelector('.list-container');
+// eslint-disable-next-line import/no-mutable-exports
```
Comment (rloterh) [inline src/createList.js]: - [ ] OPTIONAL
You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is robust code and clean.

File: src/createList.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export let lists = [];
+// eslint-disable-next-line no-unused-vars
+let index = 1;
+
+// local storage
+export const saveData = () => {
+  localStorage.setItem('dataList', JSON.stringify(lists));
+};
+
```
Comment (rloterh) [inline src/createList.js]: - [ ] Editing a list is not working as intended. When a task _one_ is added and then then renamed to _one man_, the changes do not reflect in the local storage. Kindly endeavor to improve upon this aspect.

**Editing / Update error:**
image

File: src/index.html [hunk 3]
```diff
@@ -3,13 +3,19 @@
...
     <title>To do List</title>
 </head>
 <body>
     <div class="list-items">
         <div class="to-do">Today's To Do 💪🏻</div>
         <div>
-            <input class="input-list" type="text" placeholder="Add to your list...">
+            <input class="input-list" type="text" maxlength="50" placeholder="Add to your list...">
         </div>
         <div class="list-container" >
```
Comment (rloterh) [inline src/index.html:22]: - [ ] OPTIONAL
Checked items are not removed from the list/ local storage when the _Clear all completed_ button is clicked. Kindly ensure to improve upon this aspect

File: webpack.config.js [hunk 4]
```diff
@@ -1,29 +1,28 @@
 // import HtmlWebpackPlugin from 'html-webpack-plugin';
```
Comment (rloterh) [inline webpack.config.js]: - [ ] OPTIONAL
Webpack is not configured properly and hence your application is not working as intended. You can update your webpack to look something similar to this:

```javascript
const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  mode: 'development',
  entry: './src/index.js',
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [{
      test: /\.css$/i,
      use: ['style-loader', 'css-loader'],
    }],
  },
  devServer: {
    contentBase: './dist',
  },

  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
};
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Remoz, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DanSam5K — APPROVED** [review]
# Hi @Reem-lab, :v: Approved :v: 
- [x] Good job implementing the changes required of you :heavy_check_mark: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**